### ISSUE TITLE
Yagi4 antenna files 

### ISSUE THREAD

**Issue by rlpipkin** [QDeltaSoft/qradiopredict#8]
In antenna.cxx load_NEC_antenna_pattern, the current code attempts to load a yagi4 pattern.  If possible could you include your yagi pattern and notes on how you generate new patterns?  The code is currently looking at /home/adrian/Antennas/<antenna_name>.txt

**Comment by kantooon** [comment]
Right. The antenna pattern files are just text files, 3D plots generated by 4NEC2 (you can get it working in Wine). I will include a sample file in the repo.

**Comment by kantooon** [comment]
Sorry about the delay. Almost forgot about this. I've added two sample plots generated with 4NEC2.
I actually had to go fetch these from the Flightgear data, since I have not used them in qradiopredict ever, only in Flightgear. One note to make is since usually antennas are symmetrical across one axis, the plot can be actually cut in half. Of course, if your antenna has some weird pattern, that observation no longer holds. Unfortunately 4NEC2 is a Windows thing and has to be run in Wine, I have not come across yet on a similar tool for Linux (hint, may be an avenue to wrap a GUI around NEC2).
Hope this helps.
